src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8240,9 +8242,10 @@ private StatementSyntax ParseLocalDeclarationStatement()
                         mods[i] = this.AddError(mod, ErrorCode.ERR_BadMemberFlag, mod.Text);
                     }
                 }
-
+                // PROTOTYPE
```

Comment:
>PROTOTYPE [](start = 19, length = 9)

For some reason, we typically use the branch name in PROTOTYPE markers. But, it's more convenient without. @jaredpar, OK with simple "PROTOTYPE"?